draft-iab-path-signals-collaboration.md

```diff
@@ -89,12 +89,15 @@ will be used whether intended or not. Protocol designers should
 consider either hiding these signals when the information should not
 be visible, or using explicit signals when it should be. 
 
-Both can be achieved (hiding) or supported (explicit expose) using encryption.
-While encryption is primarily deployed to protect and authenticate access to
-user data, it also supports selected exposure to those entities that are
-addressed by a certain function. As such encryption improves privacy, supports innovation
+This documents discusses principles for explicit exposure and provides guidance for 
```

Comment:
Would this replace the previous paragraph? Having both start with "this document... principles for..." is a bit redundant.